src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Classes which represent entities should not be called in a plural, so, you should call it "Color"
```suggestion
public enum Color {
```